app/models/invenio_rdm_record_converter.rb

```diff
@@ -139,7 +139,7 @@ def record_for_export
   def invenio_pids(doi)
     {
       "doi": {
-        "identifier": doi, # doi is stored in an array
+        "identifier": doi&.strip,
```

Comment:
Removing the comment since it was from when we first started on this and were not aware of how DigitalHub was storing things.

Reply:
I will make this change quickly. I did not make the change initially thinking that we had intentionally left the logic as is for some reason, but it definitely seemed suspicious.